src/tree/mod.rs

```diff
@@ -656,16 +656,15 @@ impl RatchetTree {
             self.nodes[sender_index.as_usize()] = leaf_node;
             // Check if it is a self-update
             if sender_index == self.get_own_node_index() {
```

Comment:
This is not the sender index anymore.

Reply:
What do you mean? What else would it be? I'm not changing that variable, I think.

Reply:
Well.... I don't know 😆